docs/index.md

```diff
@@ -197,14 +197,14 @@ your program. That means it's easy to process them afterwards.
 
 ```Lua
 -- You can round the animated health to whole numbers.
-local wholeHealth = Computed(function()
-	return math.round(health:get())
+local wholeHealth = Computed(function(use, scope)
+	return math.round(use(health))
 end)
 
 -- You can format it as text and put it in some UI, too.
 local myText = New "TextLabel" {
-	Text = Computed(function()
-		return "Health: " .. wholeHealth:get()
+	Text = Computed(function(use, scope)
+		return "Health: " .. use(wholeHealth)
```

Comment:
Scope shouldn't be provided as it is never used
Should be scoped
```suggestion
	Text = scope:Computed(function(use)
```

Reply:
done